### ISSUE TITLE
does unlocking double cash get u flagged?

### ISSUE THREAD

**Issue by RVG-Revenge** [MaliciousFiles/EditPlayerData#21]
does unlocking double cash get u flagged? i made a new account and edited my lvl to 140 and got all monkey knowledge and upgrades but im wondering in double cash gets u flagged.. if it does then i don't wanna risk it.. someone let me know plz

**Comment by krysta-saff** [comment]
The game gives you a basic warning about using any mods at all, but typically if it's not something that is actively affecting any leaderboards or matched co-op, they don't mind too much. I personally unlocked it and haven't had any issues, but honestly this is very much a "use at your own risk" type of thing. The game even recommends making a separate account for using mods to make sure you don't get your original or main one banned.

I will say that I had an issue a couple of days ago where I actually paid for the IAP for Fast Tracks, and somehow they didn't save my progress over that ~4-ish hour session, and I lost the Fast Tracks the next time I logged in. So I will hopefully be able to use this tool to get it back.

tl;dr Probably won't get banned, but it's recommended to just make a separate account to try it out on, and there's no 100% certainty in this sort of thing.

**Comment by MaliciousFiles** [comment]
@krysta-saff pretty much summed it up. I will say that one of the main reasons I made this mod is so that you can easily make a new mod account and get it up to speed (the other big reason being situations like @krysta-saff mentioned).

**Comment by RVG-Revenge** [comment]
@krysta-saff so u unlocked it on ur main account using the mod and u didn't get flagged?

**Comment by krysta-saff** [comment]
@RVG-Revenge  Yes, that's correct. Again, though, this is anecdotal and should be taken with a grain of salt.

Modding for pretty much any game that isn't explicitly open to it (like, Steam Workshop sorts of things) is always a risk, especially for things like this that are specifically circumventing NK's IAP's, be it 2x cash, Fast Tracks, or even just insta-levelling all of the unlocks which is also an IAP. They do work hard on the game and keep it updated despite being 6+ years old, so I would still consider supporting them if you can. (While I did use the tool for the 2x cash, I have also bought it for my little sister, so it feels more fair to me. No judgement though, I appreciate the tool providing accessibility because grinding out everything is a slog and not everyone can afford to support the devs how they want.)
